laura/portfolio/src/main/webapp/script.js

```diff
@@ -71,30 +71,35 @@ async function deleteAllComments() {
     await printServletMsg();
 }
 
-function loadMap () {
+async function loadMap () {
     const BUH = {lat: 44.43, lng: 26.10};
     const map = new google.maps.Map(
         document.getElementById('map'),
         {center: BUH, zoom: 3});
 
+   /*
     var BUHmarker = createMarker(map, 'Bucharest, Romania', BUH, 'I live here since 2000!');
-    const CONmarker = createMarker(map, 'Constanta, Romania', {lat: 44.163818, lng: 28.633451}, 'Attended a music festival.');
-    const OXFmarker = createMarker(map, 'Oxford, UK', {lat: 51.752490, lng: -1.251908}, 'I study here.');
+    const CONmarker = createMarker(map, 'Constanta, Romania', {lat: 44.163818, lng: 28.633451}, 'Attended a 4-night music festival by the sea.');
+    const OXFmarker = createMarker(map, 'Oxford, UK', {lat: 51.752490, lng: -1.251908}, 'I study here. So many great memories!');
     const VDImarker = createMarker(map, 'Val d\'Isere, France', {lat: 45.446212, lng: 6.977692}, 'I went skiing.');
     const ZRHmarker = createMarker(map, 'Zurich, Switzerland', {lat: 47.375978, lng: 8.541283}, 'Visited some friends.');
-    const Bakumarker = createMarker(map, 'Baku, Azerbaijan', {lat: 40.407327, lng: 49.854926}, 'I took part in IOI!');
-    const Bratmarker = createMarker(map, 'Bratislava, Slovakia', {lat: 48.140940, lng: 17.103918}, 'Central European Olympiad');
+    const Bakumarker = createMarker(map, 'Baku, Azerbaijan', {lat: 40.407327, lng: 49.854926}, 'I took part in IOI! Also called the City of Wind for a good reason.');
+    const Bratmarker = createMarker(map, 'Bratislava, Slovakia', {lat: 48.140940, lng: 17.103918}, 'Central European Olympiad and went rafting!');
     const STKmarker = createMarker(map, 'Stockholm, Sweden',{lat: 59.324801, lng: 18.068571}, 'A city I wanted to visit for some time. Formed from 14 islands!');
     const DOHmarker = createMarker(map, 'Doha, Qatar', {lat: 25.287784, lng: 51.530633}, 'One of the most impressive, unique and richest cities I\'ve seen! And hottest weather!');
+*/
```

Comment:
do not submit commented out code.. either add a TODO() or remove it entirely.

Reply:
done